runtime/tests/org.eclipse.core.tests.runtime/src/org/eclipse/core/tests/runtime/jobs/GithubBug_193.java

```diff
@@ -24,22 +24,18 @@
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicLong;
-import junit.framework.TestCase;
 import org.eclipse.core.runtime.IProgressMonitor;
 import org.eclipse.core.runtime.IStatus;
 import org.eclipse.core.runtime.Status;
 import org.eclipse.core.runtime.jobs.IJobChangeEvent;
 import org.eclipse.core.runtime.jobs.Job;
 import org.eclipse.core.runtime.jobs.JobChangeAdapter;
-import org.junit.FixMethodOrder;
-import org.junit.Test;
-import org.junit.runners.MethodSorters;
+import org.junit.jupiter.api.Test;
 
 /**
  * Test for bug https://github.com/eclipse-platform/eclipse.platform/issues/193
  */
-@FixMethodOrder(MethodSorters.NAME_ASCENDING)
-public class GithubBug_193 extends TestCase {
+public class GithubBug_193 {
```

Comment:
You also removed the enforced order here. Was this intentional?

Reply:
Yes, the class only contains a single test method, so there is no order that can be fixed, which makes it unnecessary to migrate the annotation to JUnit 5.

Reply:
👍 
I totally missed that, he.